docs/router-store/api.md

```diff
@@ -133,18 +139,17 @@ export interface State {
   router: RouterReducerState<RouterStateUrl>;
 }
 
+@Injectable()
 export class CustomSerializer implements RouterStateSerializer<RouterStateUrl> {
+  constructor(/* If you need to, you can inject services, too */) {}
```

Comment:
This can be removed?

Reply:
I added this to highlight that it the Serializer is a regular service and not just a mapper class. Do you think this is too obvious to mention?

Reply:
Personally, I do think it's too obvious.